aws/requests.yaml

```diff
@@ -30,7 +30,7 @@ releases:
       requests:
         - name: external-dns
           version: ">= 2.3.0"
-    - name: "> 14.0.0"
+    - name: "> 14.0.0 < 15.0.0"
```

Comment:
Which one is working? 
https://github.com/giantswarm/releases/pull/710/files#diff-8521c1dcb284f39dd6989ad25c0b64a724da61871c39dd085e985f055b01aaf9R7-R9
For better clarity, I think your changes make sense.

Reply:
One is for aws the other for azure - I have tried both with the latest app-operator now.

See the PR provided by Christian for Azure for example - https://github.com/giantswarm/releases/pull/709